core/build.rs

```diff
@@ -0,0 +1,46 @@
+use subprocess::{Popen, PopenConfig, Redirection};
+use walkdir::WalkDir;
+
+const COMPILE_SWC_PATH: &str = "src/avm2/playerglobal/compile_swc.sh";
+
+fn main() {
+    // Only rerun the build script if the AS files change -
+    // we don't want to add the entire directory, because
+    // the 'library.swf' timestamp will be updated by
+    // 'compile_swc.sh'
+    println!("cargo:rerun-if-changed={}", COMPILE_SWC_PATH);
+    for entry in WalkDir::new("src/avm2/playerglobal") {
+        let entry = entry.unwrap();
+        if let Some(ext) = entry.path().extension() {
+            if ext == "as" {
+                println!("cargo:rerun-if-changed={}", entry.path().display());
+            }
+        }
+    }
+
+    let mut compile = Popen::create(
+        &["bash", COMPILE_SWC_PATH],
```

Comment:
There's no need to explicitly mention `bash` here, because `compile_swc.sh` has a shebang, which means it can be executed directly.

Reply:
This prevents the script from starting an all on Windows (the shebang isn't recognized on Windows, but "bash" is available from a normal Command Prompt when WSL is installed). I've changed it back - even if the actual script fails (due to missing Docker, for example), the error will be caught by `core/build.rs` and execution will continue.